src/pages/Authentication/components/Input/components/Textarea/Textarea.tsx

```diff
@@ -1,104 +1,39 @@
-import { useEffect, useMemo, useRef } from 'react';
+import { useEffect } from 'react';
 import { Field, useFormikContext } from 'formik';
-import classNames from 'classnames';
-import { TokenColorsEnum } from 'pages/Authentication/enum';
-import type { TextareaDivisionType } from './types';
-import styles from './styles.module.scss';
 import { useGetNativeAuthToken } from 'hooks/useGetNativeAuthToken';
 import { FormValuesType } from '../../types';
 import { useTokenActions } from '../../hooks/useTokenActions';
 import { CopyButton } from '@multiversx/sdk-dapp/UI/CopyButton';
-
-const DEFAULT_COLOR = '#000000';
+import CodeEditor from '@uiw/react-textarea-code-editor';
```

Comment:
eslint is not working properly. this should come before relative imports by default

Reply:
to fix the `eslint` rules is not in the scope of this PR, but certainly we have to fix it